docs/openapi/index.yaml

```diff
@@ -113,6 +113,12 @@ components:
           description: The type of the activity, Training or Technical Assistance (or both)
           items:
             type: string
+        specialistNotes:
+          description: List of notes that a specliasts enters
+          $ref: '#/components/schema/nextStepNotes'
+        granteeNotes:
+          description: List of notes that a grantee enters
```

Comment:
I think this is more `List of notes/next steps that are assigned to the grantee` rather than that they enter.